packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -310,6 +321,18 @@ class ToolbarItem extends UIComponent<WithAsProp<ToolbarItemProps>> {
     _.invoke(this.props, 'onClick', e, this.props)
   }
 
+  handleWrapperClick = e => {
+    const { menu } = this.props
+    if (menu) {
+      if (doesNodeContainClick(this.menuRef.current, e, this.context.target)) {
+        this.trySetMenuOpen(false, e)
+        if (this.itemRef) {
```

Comment:
```suggestion
        if (this.itemRef.current) {
```

This condition is wrong as `itemRef` will be always object.
***
We can use use also `_.invoke()`: `_.invoke(this.itemRef.current, 'focus')`